spec/requests/client_involvement_types_controller_swagger_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "swagger_helper"
+
+RSpec.describe "ClientInvolvementTypesController", type: :request do
```

Comment:
For consistency with other renaming of top level describe block would this be more appropriate.
```suggestion
RSpec.describe "client_involvement_types", type: :request do
```